app/models/order.rb

```diff
@@ -0,0 +1,7 @@
+class Order < ApplicationRecord
+  belongs_to :customer
+  accepts_nested_attributes_for :customer
+  validates :product_name, presence: true
+  validates :product_count, presence: true
```

Comment:
I don't know why the `numericality` option is missing, but since this is a positive integer, it should have:

```suggestion
  validates :product_count, numericality: { only_integer: true, greater_than: 0 }
```